libs/design/switch/examples/src/basic-switch/basic-switch.component.html

```diff
@@ -1 +1,7 @@
-<daff-switch [checked]="checked" (toggle)="changeSwitch()">Label</daff-switch>
\ No newline at end of file
+<daff-switch [checked]="checked">Default</daff-switch>
+<daff-switch [checked]="checked" position="left">Left Label</daff-switch>
```

Comment:
Move the position examples to a separate component called `switch-label-positions` and use `<select>` to select positions.